test/source/tests/gmail.ts

```diff
@@ -253,6 +253,19 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       await pubkeyPage.waitForContent('@container-pgp-pubkey', 'Fingerprint: 50B7 A032 B5E1 FBAB 24BA B205 B362 45FD AC2F BF3D');
     }));
 
+    ava.default('mail.google.com - saving and rendering a compose draft when offline', testWithBrowser('ci.tests.gmail', async (t, browser) => {
+      const gmailPage = await openGmailPage(t, browser, '/');
+      // create compose draft
+      await gmailPage.waitAndClick('@action-secure-compose', { delay: 1 });
+      await createSecureDraft(t, browser, gmailPage, 'compose draft', { offline: true });
```

Comment:
I would like to see adding more offline drafts here, checking the list of local drafts, opening one of them and checking that the correct one is opened.
Was there a problem with that?
I guess we can make the whole `gmailPage` offline for the period of testing? 
I managed to do this in a broswer, I guess, it can be puppeteered too.

Reply:
Done 👍